src/spikeinterface/core/basesorting.py

```diff
@@ -496,6 +496,14 @@ def precompute_spike_trains(self, from_spike_vector=None):
                 for unit_id in unit_ids:
                     self.get_unit_spike_train(unit_id, segment_index=segment_index, use_cache=True)
 
+    def _custom_cache_spike_vector(self) -> None:
```

Comment:
Are you using cache as a verb here? Would not compute instead of cache makes more sense and avoid having something that has 4 verbs? 

But I do remember that you and @samuelgarcia already had a discussion about compute vs cache in some other thread.